redaxo/src/core/default.config.yml

```diff
@@ -15,13 +15,15 @@ session:
             domain: null
             secure: null
             httponly: true
+            samesite: 'Strict'
     frontend:
         cookie:
             lifetime: null
             path: null
             domain: null
             secure: null
             httponly: true
+            samesite: 'Strict'
```

Comment:
da dies nur das session cookie betrifft ist in meinen augen auch ein "Strict" im frontend ohne probleme nutzbar per default.

es mag use-cases geben in denen man will dass 3rd parties die cookies lesen dürfen, beim session cookie selbst sehe ich darin aber nur sicherheitsprobleme und keine use-cases.